claims-related/scripts/single_source_conversion_helpers.py

```diff
@@ -220,12 +221,15 @@ def merge_multiple_eligibility_options(single_source_dict):
             single_source_dict[measure_to_update]['eligibilityOptions'].append(
                 single_source_dict[measure]['eligibilityOptions'][0])
             # If the new eligibility option has new performance options as well, add those too.
-            if (
-                single_source_dict[measure]['performanceOptions'] !=
-                single_source_dict[measure_to_update]['performanceOptions']
-            ):
-                single_source_dict[measure_to_update]['performanceOptions'] += \
-                    single_source_dict[measure]['performanceOptions']
+
+            new_performance_options = [json.loads(x) for x in (
+                    set(json.dumps(x_new, sort_keys=True)
+                        for x_new in single_source_dict[measure]['performanceOptions']) -
+                    set(json.dumps(x_old, sort_keys=True)
+                        for x_old in single_source_dict[measure_to_update]['performanceOptions'])
+            )]
+
+            single_source_dict[measure_to_update]['performanceOptions'] += new_performance_options
```

Comment:
Removing duplicates from the old way we processed the file?

Reply:
Yes. The previous way the file processed checked for an exact match, and if it wasn't found, would combine the two measures. We already de-duplicate rows in the initial read, so the only way we would get duplicates is in these multipart measure definitions. It looks like the previous code expected an exact match or something totally different so it probably didn't consider a partial match. If we're uncomfortable with that level of deduplication we can change line 232 to be `single_source_dict[measure_to_update]['performanceOptions'] += single_source_dict[measure]['performanceOptions']`, but i believe the original intent here was to get the union of performance options.